samples/event_with_eventdata.json

```diff
@@ -0,0 +1,54 @@
+{
+  "Event": {
+    "#attributes": {
+      "xmlns": "http://schemas.microsoft.com/win/2004/08/events/event"
+    },
+    "EventData": {
+      "CommandLine": "",
+      "MandatoryLabel": "S-1-16-16384",
+      "NewProcessId": "0x58",
+      "NewProcessName": "Registry",
+      "ParentProcessName": "",
+      "ProcessId": "0x4",
+      "SubjectDomainName": "-",
+      "SubjectLogonId": "0x3e7",
+      "SubjectUserName": "-",
+      "SubjectUserSid": "S-1-5-18",
+      "TargetDomainName": "-",
+      "TargetLogonId": "0x0",
+      "TargetUserName": "-",
+      "TargetUserSid": "S-1-0-0",
+      "TokenElevationType": "%%1936"
+    },
+    "System": {
+      "Channel": "Security",
+      "Computer": "WIN-LL0C19JS506",
+      "Correlation": null,
+      "EventID": 4688,
+      "EventRecordID": 1,
+      "Execution": {
+        "#attributes": {
```

Comment:
Very nicely done.
Do we have a sample with attributes + content for the same element?

Reply:
added :)